### ISSUE TITLE
java.security.NoSuchAlgorithmException

### ISSUE THREAD

**Issue by xkz1994** [ikvm-revived/ikvm-maven#14]
ArtifactTransferException: Could not transfer artifact e-iceblue:spire.xls.free:jar:5.1.0 from/to central (https://repo1.maven.org/maven2/): java.security.NoSuchAlgorithmException: Error constructing implementation (algorithm: Default, provider: SunJSSE, class: sun.security.ssl.SSLContextImpl$DefaultSSLContext)

SocketException: java.security.NoSuchAlgorithmException: Error constructing implementation (algorithm: Default, provider: SunJSSE, class: sun.security.ssl.SSLContextImpl$DefaultSSLContext)
   在 org.eclipse.aether.transport.http.HttpTransporter.execute(HttpUriRequest , EntityGetter )
   在 org.eclipse.aether.transport.http.HttpTransporter.implPeek(PeekTask )
   在 org.eclipse.aether.spi.connector.transport.AbstractTransporter.peek(PeekTask task)
   在 org.eclipse.aether.connector.basic.BasicRepositoryConnector.PeekTaskRunner.runTask()
   在 org.eclipse.aether.connector.basic.BasicRepositoryConnector.TaskRunner.run()
	IKVMTest	C:\Users\xkz19\.nuget\packages\ikvm.maven.sdk\1.0.1\buildTransitive\netstandard2.0\IKVM.Maven.Sdk.targets	93

**Comment by NightOwl888** [comment]
A similar issue was previously reported here: https://github.com/ikvm-revived/ikvm/issues/79#issuecomment-1177201902. Looks like the solution is to use the latest pre-release of `IKVM` in addition to `IKVM.Maven.Sdk`.

```xml
  <ItemGroup>
    <PackageReference Include="IKVM.Maven.Sdk" Version="1.0.1" />
    <PackageReference Include="IKVM" Version="8.2.2-prerelease0553" />
  </ItemGroup>
```

**Comment by xjjius** [comment]
I tried:

<ItemGroup>		
		<PackageReference Include="IKVM.Maven.Sdk" Version="1.0.2" />
		<PackageReference Include="IKVM.NET.Sdk" Version="8.3.0-prerelease0395" />
		<PackageReference Include="IKVM" Version="8.3.0-prerelease0395" />
</ItemGroup>

same error

**Comment by wasabii** [comment]
Can you check whether this is solved in 8.4? Reopen if so. After the socket rewrite, I also fixed up a bunch of SSL related things.

Reopen if so.
